drivers/gpio/gpio_stm32.c

```diff
@@ -41,24 +45,37 @@ static void gpio_stm32_isr(int line, void *arg)
  */
 const int gpio_stm32_flags_to_conf(int flags, int *pincfg)
 {
-	int direction = flags & GPIO_DIR_MASK;
-	int pud = flags & GPIO_PUD_MASK;
 
```

Comment:
This is a code cleaning, right?
Shouldn't be put in a separate commit rather than keep it in "new GPIO API" commit?

Reply:
No this is linked to the fact I'm using `flags` below and don't need `direction` and `pud` anymore.
So directly linked to new API.